json_schemas/suppliers.json

```diff
@@ -33,12 +33,12 @@
       "items": {
         "type": "string"
       },
-      "minItems": 0
+      "minItems": 0,
+      "maxItems": 10
     }
   },
   "required": [
     "id",
-    "name",
-    "contactInformation"
```

Comment:
Looks like there are checks for this in the PUT endpoint already. And the test for missing contact information seem to pass.